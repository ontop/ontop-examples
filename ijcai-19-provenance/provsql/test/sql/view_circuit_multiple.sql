\set ECHO none
SET search_path TO public, provsql;

CREATE TABLE vc_result AS
SELECT city, view_circuit(provenance(),'d') AS ok
FROM ( 
  SELECT DISTINCT p1.city FROM
    personnel p1, personnel p2
  WHERE
    p1.id=1 AND
    p2.id=1
) t;

SELECT remove_provenance('vc_result');

SELECT city, ok FROM vc_result ORDER BY city;
DROP TABLE vc_result;
