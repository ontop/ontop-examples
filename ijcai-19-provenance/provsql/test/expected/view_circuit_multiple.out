\set ECHO none
 remove_provenance 
-------------------
 
(1 row)

   city   | ok 
----------+----
 New York | t
(1 row)

